packages/common/src/util/http.js

```diff
@@ -256,15 +256,19 @@ function encodeRequestBody(body) {
 }
 
 async function readResponseBody(response, parseAs) {
+  if (!response.body) {
```

Comment:
@hunterachieng did we resolve this properly or is it still a problem?

Reply:
@josephjclark I am unable to fix it. I have tried different ways but its failing on the contentLength